### PR TITLE
Added my photo and other fixes.

### PR DESCRIPTION
**Pull request by nzec**
**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

* Added my photo, social links and coordinates.
* Added a color scheme to the map which fits the overall theme of the website (white and grey)
* Added a nav link for the Map.

Here is a live link : https://duttaditya18.github.io/gci17.fossasia.org/
My task: https://codein.withgoogle.com/dashboard/task-instances/6285540099883008/

@fossasia/fossasia-gci Please take a look!

### CODE REVIEW

**Comment by anshumanv** [comment]
Please check the appropriate points in the PR comment.

**Comment by nzec** [comment]
@anshumanv Done!

File: _data/students.yml [hunk 1]
```diff
@@ -44,23 +44,24 @@
...
   github: saarthakchats
-  twitter: SaarthakChats41
+  twitter: SaarthakChats41 
 
 - name: Nguyen Chanh Dai
   image: ncdai.jpg
   github: ncdai3651408
   facebook: ncdai3651408
 
-- name: Kartikeya Singh
```
Comment (anshumanv) [inline _data/students.yml]: Please don't touch other students's details.

File: _data/students.yml [hunk 2]
```diff
@@ -44,23 +44,24 @@
 
 - name: Kuba Michalski
   country: Poland
-  image: kuba-michalski.jpg
+  image: kuba-michalski.jpg
```
Comment (anshumanv) [inline _data/students.yml]: Spacing not needed at the end

File: _data/students.yml [hunk 3]
```diff
@@ -44,23 +44,24 @@
...
-  image: kuba-michalski.jpg
+  image: kuba-michalski.jpg  
   github: kubami9
   twitter: top_kuba
 
 - name: Saarthak Chaturvedi
   image: Saarthak.jpg
   github: saarthakchats
-  twitter: SaarthakChats41
+  twitter: SaarthakChats41
```
Comment (anshumanv) [inline _data/students.yml]: Remove the spacing at the end

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by nzec** [comment]
@anshumanv I have done the necessary changes and squashed my commits as well.

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Please try to use your own image and wait for other mentors to review.

**Comment by nzec** [comment]
@anshumanv I am not the only one who has not used their image.

**Comment by anshumanv** [comment]
Ah @duttaditya18, no issues then. :+1:

File: index.html [hunk 4]
```diff
@@ -958,226 +961,112 @@ <h6 class="title">Quicklinks</h6>
                       lng: 0
                     },
                     styles: [{
-                      "elementType": "geometry",
```
Comment (anshumanv) [inline index.html:961]: Why did you remove this?

Comment (nzec) [reply index.html:961]: I did not remove it. I changed it. It is because they are the color theme for the map.

The map previously there had a blue color scheme which was not matching the website. So I changed it into a white and grey color scheme that now matches the website.

You can see it here : https://i.imgur.com/ZdVwqzE.png

Comment (anshumanv) [reply index.html:961]: Okay cool, nice work! :smile:

**Comment by anshumanv** [comment]
I think this is good to go, merging.

**Comment by nzec** [comment]
@anshumanv Thank you so much for helping me! I have been learning so much in these days. It is incredible!

**Comment by anshumanv** [comment]
It's good to see students like you motivated towards open source, keep up the good work! :smile:
